generated/nifake/nifake/session.py

```diff
@@ -697,8 +697,8 @@ def __init__(self, resource_name, options={}, id_query=False, reset_device=False
 
         '''
         if _grpc_channel:
-            import nifake._grpc as _grpc
-            library_interpreter = _grpc.GrpcStubInterpreter(_grpc_channel)
+            import nifake._grpc_stub_interpreter as _grpc_stub_interpreter
+            library_interpreter = _grpc_stub_interpreter.GrpcStubInterpreter(_grpc_channel)
         else:
             library_interpreter = _library_interpreter.LibraryInterpreter(encoding='windows-1251')
```

Comment:
you could move the LibraryInterpreter import in here in order to be consistent.

Reply:
In general, I keep all imports at the top, but it was important to have the grpc import here because just importing it requires the grpcio package to be installed, whereas importing the library_interpreter doesn't need anything extra.

Still, if it were only this `if`/`else` block, I'd go ahead and move it for consistency, but there's also the mako `%else` block that needs the `_library_interpreter`, so I'm going to leave the import at the top.

Reply:
To be clear, the import of gRPC must happen only inside the `if` block, by design. We do not want `nilrt` users who are just using it locally to suddenly require gRPC packages.